environments/india/terraform.yml

```
@@ -95,6 +95,36 @@ servers:
     group: celery
     os: jammy
  
+  - server_name: "celerybeat_a0--india"
```

Is the double hyphen intentionall in this name?
```suggestion
  - server_name: "celerybeat_a0-india"
```